benches/src/filter_list.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Patching HTML</title>
+		<style>
+			.items {
+				margin: 1em 0;
+				padding: 0;
+				display: flex;
+				flex-wrap: wrap;
+				gap: 2px;
+			}
+
+			.items > * {
+				display: flex;
+				align-items: center;
+				justify-content: center;
+				width: 3em;
+				height: 2em;
+				margin: 0;
+				padding: 0;
+				background: #eee;
+			}
+		</style>
+	</head>
+	<body>
+		<div id="app"></div>
+		<script type="module">
+			import { measureName, measureMemory } from './util.js';
+			import { render, createElement as h, Component } from 'framework';
+
+			function App(props) {
+				return h('div', {}, [
+					h(
+						'div',
+						{ class: 'items' },
+						props.items.map(id => h('article', { key: id }, id))
```

Comment:
Could we also add one that does `props.items.map(id => h(Row, { key: id }, id))` because this is the harder issue we're facing in diff-children